src/Misc/expressionFunc/hashFiles/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "hashFiles",
+  "version": "1.0.0",
+  "description": "GitHub Actions HashFiles() expression function",
+  "main": "lib/hashFiles.js",
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write **/*.ts",
+    "format-check": "prettier --check **/*.ts",
+    "lint": "eslint src/**/*.ts",
+    "pack": "ncc build -o ../../layoutbin/hashFiles",
+    "all": "npm run build && npm run format && npm run lint && npm run pack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/actions/runner.git"
+  },
+  "keywords": [
+    "actions"
+  ],
+  "author": "GitHub Actions",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/glob": "^0.1.0"
+  },
+  "devDependencies": {
+    "@types/node": "^12.7.12",
+    "@typescript-eslint/parser": "^2.8.0",
+    "@zeit/ncc": "^0.20.5",
+    "eslint": "^5.16.0",
+    "eslint-plugin-github": "^2.0.0",
+    "prettier": "^1.19.1",
+    "ts-jest": "^24.2.0",
```

Comment:
i think you dont need ts-jest anymore

Reply:
yep, it's copied from the actions/typescript-action-template...
i will remove this